packages/wasm-dpp/test/integration/dataContract/validation/validateDataContractFactory.spec.js

```diff
@@ -993,15 +1010,13 @@ describe('validateDataContractFactory', function main() {
         expectValidationError(result, JsonSchemaCompilationError);
 
         const [error] = result.getErrors();
-
-        expect(error.getCode()).to.equal(1004);
-
-        expect(error.message).to.be.a('string').and.satisfy((msg) => (
-          msg.startsWith('unknown format "lalala" ignored in schema')
-        ));
+// TODO
```

Comment:
Is this `todo` because something is not supported or unfinished?

Reply:
thanks, almost missed this one